art/estimators/tensorflow.py

```diff
@@ -119,3 +121,126 @@ def fit(self, x: np.ndarray, y, batch_size: int = 128, nb_epochs: int = 20, **kw
         :param nb_epochs: Number of training epochs.
         """
         NeuralNetworkMixin.fit(self, x, y, batch_size=128, nb_epochs=20, **kwargs)
+
+    def _apply_preprocessing_defences(self, x, y, fit: bool = False) -> Tuple[Any, Any]:
+        """
+        Apply all preprocessing defences of the estimator on the raw inputs `x` and `y`. This function is should
+        only be called from function `_apply_preprocessing`.
+
+        The method overrides art.estimators.estimator::BaseEstimator._apply_preprocessing_defences().
+        It requires all defenses to have a method `forward()`.
+        It converts numpy arrays to PyTorch tensors first, then chains a series of defenses by calling
```

Comment:
I think this version is for tensorflow v2, so you need to update the document to tensorflow v2. I see many other places still keep pytorch in a tensorflow document.

Reply:
Great catch! I have changed all `pytorch` references to `tensorflow` references.